forc-plugins/forc-crypto/src/keygen/new_key.rs

```diff
@@ -0,0 +1,64 @@
+//! This file is migrated from https://github.com/FuelLabs/fuel-core/blob/master/bin/keygen/src/keygen.rs
```

Comment:
Yes, it is already done https://github.com/FuelLabs/fuel-core/pull/1397, but I'm waiting for a fix to their Cargo.toml to be able to include the code directly. Until that happens it will be copied. I have a follow up ticket already #5176 5170